### ISSUE TITLE
Overview page bookings backend

### ISSUE THREAD

**Issue by qy21gafy** [amosproj/amos2022ws05-shared-desk-mgmt#40]
## User story
1. As a user
2. I want / need to see my recent bookings in the system on my overview page
3. So that I can view them quickly when I login in

## Acceptance criteria
* Implement end point that gives back the recent bookings of the current user
* Backend part

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by qy21gafy** [comment]
The most recent bookings should contain the 10 most recent bookings including the desk name, building, room, date and time.
